src/components/pages/ExplorePage/ExplorePage.tsx

```diff
@@ -172,10 +172,12 @@ const recommendedPoolsFetcher = async (
   }
 
   // reduce poolsMap to only pools which were recommended
-  const recommendedPoolDataMap: RecommendedPoolDataMap = {};
-  Object.values(map).forEach(({ poolId }) => {
-    recommendedPoolDataMap[poolId] = poolsMap[poolId];
-  });
+  const recommendedPoolDataMap = Object.values(
+    map
+  ).reduce<RecommendedPoolDataMap>((acc, { poolId }) => {
+    acc[poolId] = poolsMap[poolId];
+    return acc;
+  }, {});
```

Comment:
I noticed we don't have `lodash` as a dep, but if you're open to adding it we can make this a two-liner with [pick](https://lodash.com/docs/4.17.15#pick):

```ts
import { pick } from 'lodash';

const recommendedPoolIds = Object.values(map).map(({ poolId }) => poolId);
const recommendedPoolDataMap = pick(poolsMap, recommendedPoolIds);
```